packages/ra-core/src/core/Resource.tsx

```diff
@@ -15,46 +14,19 @@ export const Resource = (props: ResourceProps) => {
         <ResourceContextProvider value={name}>
             <Routes>
                 {create && (
-                    <Route
-                        path="create/*"
-                        element={
-                            <CanAccess action="create" resource={name}>
```

Comment:
I'd remove the CanAccess wrapper only when the page controllers do the access control themselves. Otherwise it breaks unit tests.

Reply:
You're right, even though we will do it, it has nothing to do with this PR. I'll revert it